src/backend/utils/mmgr/vmem_tracker.c

```diff
@@ -673,6 +722,10 @@ VmemTracker_RegisterStartupMemory(int64 bytes)
 	pg_atomic_add_fetch_u32((pg_atomic_uint32 *) segmentVmemChunks,
 							startupChunks);
 
+#ifdef FAULT_INJECTOR
+	SIMPLE_FAULT_INJECTOR("vmem_oom_startup");
+#endif
+
```

Comment:
```suggestion
	SIMPLE_FAULT_INJECTOR("vmem_oom_startup");
```

`SIMPLE_FAULT_INJECTOR()` expands to nothing if `FAULT_INJECTOR ` isn't enabled, so we don't need to guard it with the macro.

Reply:
OK, I'll remove it later.